terraform/bod_lambdas.tf

```diff
@@ -1,23 +1,9 @@
-# IAM assume role policy document for the roles we're creating for the
-# lambda functions
-data "aws_iam_policy_document" "lambda_assume_role_doc" {
-  statement {
-    effect = "Allow"
-
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "Service"
-      identifiers = ["lambda.amazonaws.com"]
-    }
-  }
-}
-
 # The roles we're creating for the lambda functions
```

Comment:
Capitalization - there are probably a bunch of other similar cases if you feel like fixing them:
```suggestion
# The roles we're creating for the Lambda functions
```

Reply:
Please see [021f2d9](https://github.com/cisagov/cyhy_amis/pull/410/commits/021f2d999cfd1d358eeec475db917ee0507cd8e2).